### PR TITLE
Add Fire Emblem Echoes: Shadows of Valentia

### PR DESCRIPTION
**Pull request by Rignak**
Adds Fire Emblem Echoes: Shadows of Valentia.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/fire-emblem-echoes-shadows-of-valentia/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Fire Emblem Echoes: Shadows of Valentia"
+description = "Fire Emblem Echoes: Shadows of Valentia is a tactical role-playing game developed by Intelligent Systems and published by Nintendo for the Nintendo 3DS handheld video game console in 2017. The game is a full remake of the 1992 Famicom title Fire Emblem Gaiden, the second entry in the Fire Emblem series."
+needs_system_files = false
+needs_shared_font = false
+
+[[releases]]
```
Comment (TheKoopaKingdom) [inline games/fire-emblem-echoes-shadows-of-valentia/game.dat]: Add a USA release, if this game has one.

File: games/fire-emblem-echoes-shadows-of-valentia/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+title = "00040000001B4100"
+region = "eur"
+release_date = "2017-04-20"
```
Comment (Leo626) [inline games/fire-emblem-echoes-shadows-of-valentia/game.dat]: Wikipedia says the eur release was on 19th.

File: games/fire-emblem-echoes-shadows-of-valentia/game.dat [hunk 3]
```diff
@@ -0,0 +1,19 @@
+title = "Fire Emblem Echoes: Shadows of Valentia"
+description = "Fire Emblem Echoes: Shadows of Valentia is a tactical role-playing game developed by Intelligent Systems and published by Nintendo for the Nintendo 3DS handheld video game console in 2017. The game is a full remake of the 1992 Famicom title Fire Emblem Gaiden, the second entry in the Fire Emblem series."
+needs_system_files = false
+needs_shared_font = false
+
```
Comment (Hexagon12) [inline games/fire-emblem-echoes-shadows-of-valentia/game.dat]: Would be nice if you included a list of issues that happen with
`github_issues = [issue numbers here]`

Comment (chris062689) [reply games/fire-emblem-echoes-shadows-of-valentia/game.dat]: Agreed.

**Comment by Hexagon12** [comment]
Closed due to inactivity.

**Pull request closed without merging**